composer.json

```diff
@@ -29,7 +29,7 @@
         "phpunit/phpunit": "^8.4",
         "squizlabs/php_codesniffer": "^3.5",
         "mockery/mockery": "^1.3",
-        "vimeo/psalm": "^3.7"
+        "vimeo/psalm": ">=3.10"
```

Comment:
I'm curious why you switched from `^ `to `>=`?

Reply:
Psalm renamed `level` to `errorLevel` in 3.10 and during CI builds, we're running `prefer-lowest` on `composer update`, which was pulling down earlier versions of Psalm and causing builds to break because `errorLevel` isn't in the XSD.